infrastructure/transform/main.tf

```diff
@@ -65,6 +65,45 @@ resource "azurerm_data_factory" "adf" {
   }
 }
 
+
+resource "azurerm_data_factory_pipeline" "pipeline" {
+  name            = "databricks-pipeline-${var.naming_suffix}"
+  data_factory_id = azurerm_data_factory.adf.id
+  activities_json = <<JSON
+[
+  {
+    "name": "DatabricksPythonActivity",
+    "type": "DatabricksSparkPython",
+    "typeProperties": {
+        "pythonFile": "dbfs:/entrypoint.py",
```

Comment:
Where is this uploaded?

Reply:
I have now added and tested resource block which uploads entry point to dbfs, at the bottom of main.tf.